packages/c-radio/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { CRadioGroup, type CRadioGroupProps } from "./c-radio-group"
+export { CRadio, type CRadioProps } from "./c-radio"
+export type { RadioGroupContext } from "./radio-context"
```

Comment:
Let's keep the `C${COMPONENT_NAME}Context` naming convention here. Would like all types to be exposed with the `C` in front of it.